docs/reinstallation/README.md

```diff
@@ -0,0 +1,383 @@
+Use one of the following methods:
+
+- **GitOps Approach**: Commit and push the updated `ClusterInstance` resource to your Git repository.
+- **Manual Application**: Apply the changes directly on the hub cluster:
+
+  ```sh
+  oc apply -f clusterinstance-example.yaml  
+  ```  
+
+### 4. Optional. Monitoring the cluster reinstallation
+
+The reinstallation has two phases:
+
+- **Phase 1**: Reinstallation request handling
+  - **Request validation**: The SiteConfig operator validates the request.
+  - **Data preservation**: The SiteConfig operator backs up labeled resources.
+  - **Cleanup**: The SiteConfig operator removes existing installation manifests. If this step times out, the reinstallation stops and the `ClusterInstance` resource is paused.
+  - **Data restoration**: The SiteConfig operator restores preserved data.
+
+- **Phase 2**: Cluster provisioning
+  - **Manifest regeneration**: The SiteConfig operator generates new manifests from templates.
+  - **Cluster installation**: The cluster is provisioned using the new manifests.
+
+You can track progress in the `status.reinstall.conditions` and `status.conditions` fields for phase 1 and 2, respectively. To track the cluster reinstallation progress, the SiteConfig operator provides the following status conditions:
+
+- `ReinstallRequestProcessed` – Indicates the overall status of the reinstallation request.
+- `ReinstallRequestValidated` – Confirms the validity of the reinstallation request.
+- `ReinstallPreservationDataBackedUp` – Tracks the backup status of preserved data.
+- `ReinstallClusterIdentityDataDetected` – Determines whether cluster identity data is available for preservation.
+- `ReinstallRenderedManifestsDeleted` – Monitors the deletion of rendered manifests associated with the `ClusterInstance` resource.
+- `ReinstallPreservationDataRestored` – Tracks the restoration status of preserved data.
+
+The`status.reinstall` field provides further information about the reinstallation process with the following fields:
+
+- `InProgressGeneration` – Identifies the active generation being processed for reinstallation.
+- `ObservedGeneration` – Indicates the last successfully processed reinstallation request.
+- `RequestStartTime` – Indicates the time when the reinstallation request was initiated.
+- `RequestEndTime` – Indicates the time when the reinstallation process was completed.
+- `History` – Displays past reinstallation attempts, including generation details, timestamps, and specification changes to the `ClusterInstance` resource.
+
+For more information about the reinstallation conditions, see [Cluster reinstallation status conditions](status_conditions.md).
+
+To monitor the cluster reinstallation progress, complete the following steps:
+
+1. Verify that the reinstallation request is being processed:  
+
+    ```sh
+    oc get clusterinstance clusterinstance-example -n some-namespace -o json | jq -r '.status.reinstall.conditions[] | select(.type=="ReinstallRequestProcessed")'
```

Comment:
Can use the oc tools 'custom-columns' to avoid dependence on the external jq tool (here and others):

```suggestion
    oc get clusterinstance cnfdf02 -o custom-columns=STATUS:'.status.conditions[?(@.type=="ReinstallRequestProcessed")].status'
```